components/bpmn/org.wso2.carbon.bpmn.analytics.publisher/src/main/java/org/wso2/carbon/bpmn/analytics/publisher/utils/BPMNDataReceiverConfig.java

```diff
@@ -105,7 +105,6 @@ public static String getPassword() throws RegistryException, UserStoreException
                         base64DecodeAndDecrypt(encryptedPassword);
                 password = new String(decryptedPassword);
```

Comment:
We should have to store it into a String, because when we create a Datapublisher object we must have to pass the password as a  String format. I created a jira for that in DAS. 
https://wso2.org/jira/browse/DAS-353